docs/Using-Orion/Privacy-Groups.md

```diff
@@ -0,0 +1,17 @@
+# Privacy Groups 
+
+When [transactions are sent](../Reference/API-Methods.md#send) to: 
+
+* List of recipients, Orion creates a privacy group containing the recipients 
+* Privacy group, the transaction is distributed to all members of the privacy group. 
+
+Pantheon maintains a [private state per privacy group](https://docs.pantheon.pegasys.tech/en/stable/Privacy/Privacy-Overview/#privacy-groups). 
+
+Endpoints are provided to create and manage privacy groups: 
```

Comment:
We can only create and delete privacy groups for now and not manage.

Reply:
Fixed